sections/all_map_locations.tex

```diff
@@ -310,8 +311,8 @@ \subsection*{Other fields}
 \end{figure}
 \vfill
 
-\begin{figure*}[!hb]
+\begin{scaledfigure}[blanker]
   \centering
-  \includegraphics[width=\linewidth]{\art/gold_dragon.jpg}
-\end{figure*}
-\vfill
+  \includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/gold_dragon.jpg}
+\end{scaledfigure}
+\vfill
```

Comment:
One more newline lurking here.

Reply:
Argh, sorry, those were actually errors fixed. See what lack of linters does to people 🤦🏻 I made a pull request with linters.